### PR TITLE
[PS-74] Fix user authentication state checks

### PR DESCRIPTION
**Pull request by eliykat**
## Type of change

- [x] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Consequential changes from https://github.com/bitwarden/jslib/pull/721.
## Code changes

* use new authService.authStatus() method

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/app/services/services.module.ts [hunk 1]
```diff
@@ -3,7 +3,6 @@ import { ToastrModule } from "ngx-toastr";
 
 import {
   JslibServicesModule,
-  WINDOW,
```
Comment (djsmith85) [inline src/app/services/services.module.ts:6]: Why are we removing this? Doesn't seem related to the proposed changes.

Comment (eliykat) [reply src/app/services/services.module.ts:6]: It's an unused import - just cleaning it up while I'm here.

**Review by djsmith85 — APPROVED** [review]
